From 022cf2558c34ff7d83e89e05718cc9e25d6b4d96 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 18 Sep 2014 14:20:11 -0400
Subject: [fs] xprtrdma: Rename frmr_wr

Message-id: <1411050057-12417-49-git-send-email-steved@redhat.com>
Patchwork-id: 96216
O-Subject: [RHEL7.1 PATCH 48/94] xprtrdma: Rename frmr_wr
Bugzilla: 1111708
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>

From: Chuck Lever <chuck.lever@oracle.com>

Clean up: Name frmr_wr after the opcode of the Work Request,
consistent with the send and local invalidation paths.

Signed-off-by: Chuck Lever <chuck.lever@oracle.com>
Tested-by: Steve Wise <swise@opengridcomputing.com>
Tested-by: Shirley Ma <shirley.ma@oracle.com>
Tested-by: Devesh Sharma <devesh.sharma@emulex.com>
Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
Upstream Id: f590e878c52c38046fd7cfa5a742ddae68717484
Bugzilla: 1111708
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/sunrpc/xprtrdma/verbs.c b/net/sunrpc/xprtrdma/verbs.c
index 08c9235..80c0163 100644
--- a/net/sunrpc/xprtrdma/verbs.c
+++ b/net/sunrpc/xprtrdma/verbs.c
@@ -1687,7 +1687,7 @@ rpcrdma_register_frmr_external(struct rpcrdma_mr_seg *seg,
  struct rpcrdma_mw *mw = seg1->mr_chunk.rl_mw;
  struct rpcrdma_frmr *frmr = &mw->r.frmr;
  struct ib_mr *mr = frmr->fr_mr;
- struct ib_send_wr frmr_wr, *bad_wr;
+ struct ib_send_wr fastreg_wr, *bad_wr;
  u8 key;
  int len, pageoff;
  int i, rc;
@@ -1721,15 +1721,15 @@ rpcrdma_register_frmr_external(struct rpcrdma_mr_seg *seg,
 
  frmr->fr_state = FRMR_IS_VALID;
 
- memset(&frmr_wr, 0, sizeof frmr_wr);
- frmr_wr.wr_id = (unsigned long)(void *)mw;
- frmr_wr.opcode = IB_WR_FAST_REG_MR;
- frmr_wr.wr.fast_reg.iova_start = seg1->mr_dma;
- frmr_wr.wr.fast_reg.page_list = frmr->fr_pgl;
- frmr_wr.wr.fast_reg.page_list_len = page_no;
- frmr_wr.wr.fast_reg.page_shift = PAGE_SHIFT;
- frmr_wr.wr.fast_reg.length = page_no << PAGE_SHIFT;
- if (frmr_wr.wr.fast_reg.length < len) {
+ memset(&fastreg_wr, 0, sizeof(fastreg_wr));
+ fastreg_wr.wr_id = (unsigned long)(void *)mw;
+ fastreg_wr.opcode = IB_WR_FAST_REG_MR;
+ fastreg_wr.wr.fast_reg.iova_start = seg1->mr_dma;
+ fastreg_wr.wr.fast_reg.page_list = frmr->fr_pgl;
+ fastreg_wr.wr.fast_reg.page_list_len = page_no;
+ fastreg_wr.wr.fast_reg.page_shift = PAGE_SHIFT;
+ fastreg_wr.wr.fast_reg.length = page_no << PAGE_SHIFT;
+ if (fastreg_wr.wr.fast_reg.length < len) {
   rc = -EIO;
   goto out_err;
  }
@@ -1738,13 +1738,13 @@ rpcrdma_register_frmr_external(struct rpcrdma_mr_seg *seg,
  key = (u8)(mr->rkey & 0x000000FF);
  ib_update_fast_reg_key(mr, ++key);
 
- frmr_wr.wr.fast_reg.access_flags = (writing ?
+ fastreg_wr.wr.fast_reg.access_flags = (writing ?
     IB_ACCESS_REMOTE_WRITE | IB_ACCESS_LOCAL_WRITE :
     IB_ACCESS_REMOTE_READ);
- frmr_wr.wr.fast_reg.rkey = mr->rkey;
+ fastreg_wr.wr.fast_reg.rkey = mr->rkey;
  DECR_CQCOUNT(&r_xprt->rx_ep);
 
- rc = ib_post_send(ia->ri_id->qp, &frmr_wr, &bad_wr);
+ rc = ib_post_send(ia->ri_id->qp, &fastreg_wr, &bad_wr);
  if (rc) {
   dprintk("RPC:       %s: failed ib_post_send for register,"
    " status %i\n", __func__, rc);
-- 
1.7.1